scripts/save_docker_cache.sh

```diff
@@ -24,6 +24,7 @@ set -o pipefail
 if [[ ${TRAVIS_BRANCH} == "master" ]] &&
     [[ ${TRAVIS_PULL_REQUEST} == "false" ]]; then
   mkdir -p $(dirname ${DOCKER_CACHE_FILE});
-  docker save $(docker history -q cartographer_fetch:${ROS_RELEASE} |
-      grep -v '<missing>') | gzip > ${DOCKER_CACHE_FILE};
+  IMAGE_NAMES=$(docker history -q cartographer_fetch:${LSB_RELEASE} | grep -v '<missing>')
```

Comment:
Should this be `ROS_RELEASE` instead of `LSB_RELEASE`? I think we use `LSB_RELEASE` only for the cartographer repo.